.github/workflows/go-lint.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
                   go-version-file: go.mod
 
             - name: Run GolangCI-Lint Scanner
-              uses: golangci/golangci-lint-action@v5
+              uses: golangci/golangci-lint-action@v6
               with:
                   # Required: the version of golangci-lint is required and must be specified without patch version.
                   version: v1.57
```

Comment:
you can update this to `v1.59` too. Another update you can make is to `commitlint.config.mjs` like https://github.com/indykite/jarvis-proto/commit/020f59d13e07b3bbffdf8b12cd23a53522426a27#diff-de210b7edf536f1e604277c7555f8091f81a9b8af754216566a4d1973becb943

Reply:
thanks, done